src/main/java/com/spikes2212/util/Limelight.java

```diff
@@ -0,0 +1,175 @@
+package com.spikes2212.util;
+
+import com.spikes2212.dashboard.Namespace;
+import com.spikes2212.dashboard.RootNamespace;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+import java.util.function.Supplier;
+
+/**
+ * This class is a limelight wrapper that includes a function that calculates the distance of the limelight from a target
+ * using the autofocus properties of the limelight.
+ *
+ * @author Yotam Yizhar
+ */
+
+public class Limelight {
+
+    private static RootNamespace rootNamespace = new RootNamespace("Limelight Values");
+    private static Namespace ConstantNamespace = rootNamespace.addChild("Constants");
+    private static NetworkTableInstance table;
+    private static double focusLength;
```

Comment:
add documentation, make this final, and put a number in it. this is true for all the Limelights in the world